hsingyuc/trashaholic
# Change markers

**Feature**
Change markers style

**Screenshot**
<img width="415" alt="螢幕快照 2020-04-13 下午4 49 44" src="https://user-images.githubusercontent.com/56378160/79125438-db915780-7da6-11ea-80ed-dce839f8ea67.png">

**Testing**
1. Click the Upper right corner Map icon button on the table page and check if markers render on the map.
1. Click each marker to check if the info card shows up.
